tests/ui/test_search.py

```diff
@@ -53,8 +58,87 @@ def test_sorting_by(base_url, selenium, category, sort_attr):
 def test_incompative_extensions_show_as_incompatible(base_url, selenium):
     page = Home(selenium, base_url).open()
     term = 'Ui-Addon-Android'
-    items = page.search_for(term)
+    items = page.search.search_for(term)
     for item in items.result_list.extensions:
         if term == item.name:
             detail_page = item.click()
             assert detail_page.is_compatible is False
+
+
+@pytest.mark.nondestructive
+def test_search_suggestion_term_is_higher(base_url, selenium):
+    page = Home(selenium, base_url).open()
+    term = 'Ui-Addon'
+    items = page.search.search_for(term, execute=False)
+    for count, item in enumerate(items):
+        if item.name == term:
+            assert count < 1
+
+
+@pytest.mark.nondestructive
+def test_search_suggestion_term_loads_correct_suggestions(base_url, selenium):
+    page = Home(selenium, base_url).open()
+    term = 'Ui-Addon'
+    items = page.search.search_for(term, execute=False)
+    assert term == items[0].name
+    term = '-Install'
+    items = page.search.search_for(term, execute=False)
+    # Sleep to let autocomplete update.
+    time.sleep(2)
```

Comment:
I wonder if `0.8` or so would suffice? Autocomplete should be fairly fast.

Reply:
Worth a shot!

Reply:
Wanna try this out still?